internal/app/machined/pkg/system/services/kubelet.go

```diff
@@ -66,22 +133,22 @@ func (k *Kubelet) Runner(config config.Configurator) (runner.Runner, error) {
 		ProcessArgs: []string{
 			"/hyperkube",
 			"kubelet",
-			"--bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf",
-			"--kubeconfig=/etc/kubernetes/kubelet.conf",
-			"--config=/var/lib/kubelet/config.yaml",
+			"--bootstrap-kubeconfig=" + constants.KubeletBootstrapKubeconfig,
+			"--kubeconfig=/etc/kubernetes/kubeconfig-kubelet",
 			"--container-runtime=remote",
-			"--runtime-request-timeout=15m",
 			"--container-runtime-endpoint=unix://" + constants.ContainerdAddress,
+			"--anonymous-auth=false",
+			"--cert-dir=/var/lib/kubelet/pki",
+			"--client-ca-file=" + constants.KubernetesCACert,
+			"--cni-conf-dir=/etc/cni/net.d",
+			"--cluster-domain=cluster.local",
+			"--pod-manifest-path=/etc/kubernetes/manifests",
+			"--rotate-certificates",
```

Comment:
//TODO check to see that we have ca key before enabling auto-certificate rotation